sway-core/src/asm_generation/fuel/functions.rs

```diff
@@ -774,18 +772,16 @@ impl<'ir, 'eng> FuelAsmBuilder<'ir, 'eng> {
             .locals_ctxs
             .pop()
             .expect("Calls guaranteed to save locals context.");
-        if locals_size != 0 {
-            if locals_size > compiler_constants::TWENTY_FOUR_BITS {
-                todo!("Enormous stack usage for locals.");
-            }
-            self.cur_bytecode.push(Op {
-                opcode: Either::Left(VirtualOp::CFSI(VirtualImmediate24 {
-                    value: locals_size as u32,
-                })),
-                comment: format!("free {locals_size} bytes for locals"),
-                owning_span: None,
-            });
+        if locals_size > compiler_constants::TWENTY_FOUR_BITS {
+            todo!("Enormous stack usage for locals.");
         }
+        self.cur_bytecode.push(Op {
+            opcode: Either::Left(VirtualOp::CFSI(VirtualImmediate24 {
+                value: locals_size as u32,
```

Comment:
Might as well remove this `as` cast, it's better practice to use `From`/`Into` and explicit the handling.